test/AspNet.Security.OAuth.Providers.Tests/WorkWeixin/WorkWeixinTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace AspNet.Security.OAuth.WorkWeixin
+{
+    public class WorkWeixinTests : OAuthTests<WorkWeixinAuthenticationOptions>
+    {
+        public WorkWeixinTests(ITestOutputHelper outputHelper)
+        {
+            OutputHelper = outputHelper;
+        }
+
+        public override string DefaultScheme => WorkWeixinAuthenticationDefaults.AuthenticationScheme;
+
+        protected internal override void RegisterAuthentication(AuthenticationBuilder builder)
+        {
+            builder.AddWorkWeixin(options => ConfigureDefaults(builder, options));
+        }
+
+        [Theory]
+        [InlineData(ClaimTypes.NameIdentifier, "my-user-id")]
+        [InlineData(ClaimTypes.Name, "wql")]
+        [InlineData("urn:workweixin:mobile", "888888")]
+        [InlineData(ClaimTypes.Email, "wql1994@126.com")]
```

Comment:
There are 7 claims mapped by default, so there should be a test case for each one.